src/chaat/handler.clj

```diff
@@ -0,0 +1,22 @@
+(ns chaat.handler
+  (:require [ring.util.response :as res]
+            [clj-time.local :as l]))
+
+(defn home
+  [request]
+  (res/response "Welcome to chaat"))
+
+(defn health-check
+  [request]
+  (->> (l/local-now)
```

Comment:
Prefer using threading to describe a sequence of operations to a singular data structure.

```clojure

(->> cart
 (add-items items)
 (apply-discount-code code)
 (apply-shipping-price address)
 (apply-taxes pincode))
```

I wouldn't use threading here for this.

Reply:
Hmm, this makes sense. I've modified this to describe a sequence (only one here) of operations to a singular data structure.

Reply:
Rebased a previous commit to include this refactor.